src/modules/auth/auth.service.ts

```diff
@@ -14,13 +18,24 @@ export class AuthService {
 
 	async validateUser(email: string, pass: string): Promise<Omit<User, "passwordHash"> | null> {
 		const user = await this.userService.handleGetUserByEmail(email);
-		const isPasswordValid = user && (await bcrypt.compare(pass, user.passwordHash));
+		const isDefaultUser = !isGoogleUser(user);
+
+		// Check if the user was created using google
+		if (user && !isDefaultUser)
+			// If so throw an error and tell user to user google authorization
+			throwHttpException(
+				RESPONSE_TYPES.CONFLICT,
+				"It looks like you signed up using Google. Please use Google to log in."
+			);
+
+		const isPasswordValid =
+			user && isDefaultUser && (await bcrypt.compare(pass, user.passwordHash));
```

Comment:
Checking if user exists and is the user is a default user, has nothing to do with password validity as far as I understand.